### DISCUSSION TITLE
Is there a way to set the speed of each specific node?

### DISCUSSION THREAD

**Discussion by talianagisan** [KillBait/AutoDrive_Course_Editor#39] [Q&A] [answered]
So the editor is nice, I can see myself using it to help make routes for most of the map. Is there a way to set it so speed of each node is changed? The speed limit on some roads is 45 but others its 30 and I want to reflect this so i dont have to set a universal 30mph.

**Comment by KillBait** [comment, marked as answer]
Unfortunately no, AutoDrive nodes are pure co-ordinates, unlike course play that stores speed aswell

**Reply by talianagisan** [reply]
Thank you for getting back to me. Unfortunate but I understand.
